package.json

```diff
@@ -2,29 +2,50 @@
   "name": "@aeternity/aeproject",
   "version": "4.6.1",
   "description": "aeternity smart contract testing tool",
-  "main": "src/lib/index.js",
+  "main": "./.build/esm/src/lib/index.js",
+  "types": "./.build/types/src/lib/index.d.ts",
   "bin": {
-    "aeproject": "./src/cli.js"
+    "aeproject": "./.build/esm/src/cli.js"
+  },
+  "exports": {
+    ".": {
+      "types": "./.build/types/src/lib/index.d.ts",
+      "require": "./.build/cjs/src/lib/index.js",
+      "import": "./.build/esm/src/lib/index.js",
+      "default": "./.build/esm/src/lib/index.js"
+    }
   },
   "scripts": {
+    "build": "tsc -b ./tsconfig.cjs.json ./tsconfig.esm.json ./tsconfig.types.json && npm run copy-files",
+    "build:clean": "rm -rf ./.build",
```

Comment:
Sometimes there are issues with `rm` e.g. on Windows, so nodejs projects that need to work on all platforms often use [rimraf](https://www.npmjs.com/package/rimraf) instead

Reply:
aeproject only lists support on WSL 2, so that shouldn't be a problem, thanks for your feedback!